sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -324,12 +331,16 @@ private[ui] class ExecutionPagedTable(
 
             <th>
               <a href={headerLink}>
-                {header}
+                <span data-toggle="tooltip" data-placement="top" title={headerTooltip}>
```

Comment:
If `headerToolTip` is empty, do we need to add this?

Reply:
Yes, but if it's empty then no tooltips popup will display.